CMakeLists.txt

```diff
@@ -173,4 +173,39 @@ endif(HYDROCHRONO_ENABLE_TESTS)
 # Not a good idea to copy DLLs
 # dosen't work on Linux
 #set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR})
-#add_DLL_copy_command("${CHRONO_DLLS}")
\ No newline at end of file
+#add_DLL_copy_command("${CHRONO_DLLS}")
+
```

Comment:
Missing include(GNUInstallDirs) to be able to use ${CMAKE_INSTALL_LIBDIR}

Reply:
Added